### DISCUSSION TITLE
Release new version with nodejs type pollution fixes

### DISCUSSION THREAD

**Discussion by jkieboom** [mswjs/msw#1439] [General]
Would it be possible to release a new version of strict-event-emitter, @mswjs/interceptors and msw to include the fixes to avoid the nodejs type pollution. Thanks!

**Comment by kettanaito** [comment]
Hey, @jkieboom. I believe all those have been released, if the fixes were there. Re-installing `msw` should automatically propagate the fixes per semver.

I don't see any changes regarding Node.js types in @mswjs/interceptors [Releases](https://github.com/mswjs/interceptors/releases). I can also notice that `strict-event-emitter` CI is failing because it misses `@types/jest` since we've set `types: []` explicitly ([here](https://github.com/open-draft/strict-event-emitter/actions/runs/3218334426/jobs/5385976574)). That seems to be not suitable for a solution at the current state. 

Also, it's worth mentioning that #1436 will likely bring in the entire `lib.dom` to MSW, which will always be used whenever you're using the library to get type definitions for Request, Response, ReadableStream, and many others. Those don't exist on older versions of Node. Those also cannot be granularly imported from `lib.dom` (or any lib at this point) because TypeScript doesn't allow that.

**Comment by jkieboom** [comment]
Afaik, my last fix to `strict-event-emitter` has not been released? We can just set `types: ["@types/jest"]` if that's an issue.

**Reply by kettanaito** [reply]
Thanks. I've added that change and released the package https://github.com/open-draft/strict-event-emitter/releases/tag/v0.2.8

**Reply by jkieboom** [reply]
Awesome, thanks!
